contrib/extractor.bat

```diff
@@ -3,57 +3,99 @@ CLS
 
 :MENU
 ECHO.
-ECHO ...............................................
-ECHO Trinitycore dbc/db2, maps, vmaps, mmaps extractor
-ECHO ...............................................
-ECHO PRESS 1, 2, 3 OR 4 to select your task, or 5 to EXIT.
-ECHO ...............................................
+ECHO ====================== Extractor ========================
+ECHO Trinitycore cameras, dbc, maps, vmaps and mmaps extractor
+ECHO =========================================================
 ECHO.
+ECHO ======================================================
 ECHO WARNING! when extracting the vmaps extractor will
 ECHO output the text below, it's intended and not an error:
-ECHO ..........................................
+ECHO ======================================================
 ECHO Extracting World\Wmo\Band\Final_Stage.wmo
 ECHO No such file.
 ECHO Couldn't open RootWmo!!!
 ECHO Done!
-ECHO ..........................................
+ECHO ======================================================
 ECHO.
-ECHO Press 1, 2, 3 or 4 to start extracting or 5 to exit.
-ECHO 1 - Extract base files (NEEDED) and cameras.
-ECHO 2 - Extract vmaps (needs maps to be extracted before you run this) (OPTIONAL, highly recommended)
-ECHO 3 - Extract mmaps (needs vmaps to be extracted before you run this, may take hours) (OPTIONAL, highly recommended)
-ECHO 4 - Extract all (may take hours)
-ECHO 5 - EXIT
+ECHO 1: Extract base files (NEEDED) and cameras.
+ECHO 2: Extract vmaps (needs maps to be extracted before you run this) (OPTIONAL, but highly recommended)
+ECHO 3: Extract mmaps (needs vmaps to be extracted before you run this, may take hours) (OPTIONAL, but highly recommended)
+ECHO 4: Extract all (may take hours)
+ECHO 5: Extract all with logs (may take hours)
+ECHO 6: EXIT
 ECHO.
-SET /P M=Type 1, 2, 3, 4 or 5 then press ENTER:
+SET /P M=Type 1, 2, 3, 4, 5 or 6 then press ENTER:
 IF %M%==1 GOTO MAPS
 IF %M%==2 GOTO VMAPS
 IF %M%==3 GOTO MMAPS
 IF %M%==4 GOTO ALL
-IF %M%==5 GOTO :EOF
+IF %M%==5 GOTO ALLWITHLOGS
+IF %M%==6 GOTO :EOF
 
 :MAPS
+ECHO.
 start /b /w mapextractor.exe
+ECHO Cameras, dbc and maps extracted.
+PING 127.0.0.1 -n 5 >NUL
```

Comment:
wtf?

Reply:
Lets the user know the process has finished. The ping just waits five seconds before moving onto the next process.
Purely cosmetic.

Reply:
just use timeout then
![obraz](https://github.com/TrinityCore/TrinityCore/assets/297439/c062ec28-fdb5-43ce-823c-c3bffa9fc62e)

Reply:
then use timeout.